include/ebpf_store_helper.h

```diff
@@ -4,13 +4,11 @@
 #pragma once
 
 #include "ebpf_program_types.h"
-#include "ebpf_shared_framework.h"
 #include "ebpf_windows.h"
 #ifdef USER_MODE
 #include "ebpf_utilities.h"
```

Comment:
ebpf_utilities also cannot be included from outside eBPF. I forget why, and ideally we should first ensure eBPF has adequate tests to discover the problem before fixing it.